File: src/leetcode/python/easy/FindTheDifference_389.py
class FindTheDifference_389(object):
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        import collections
        c = collections.Counter(s)
        for ch in t:
            c[ch] -= 1
            if c[ch] < 0:
                return ch
        return '\u0000'

File: src/leetcode/java/FindTheDifference_389.java
package leetcode.java;

public class FindTheDifference_389 {
  public char findTheDifference(String s, String t) {
    int[] ch = new int[256];
    for (char c : s.toCharArray()) ch[c] += 1;
    char car = '\u0000';
    for (char c : t.toCharArray()) {
      ch[c]--;
      if (ch[c] < 0) {
        car = c;
        break;
      }
    }
    return car;
  }
}
